### PR TITLE
commit label updates #3046

### PR DESCRIPTION
**Pull request by salomon-j**

### CODE REVIEW

File: grails-app/views/project/meriPlan/_extendedBaselineMonitoring.gsp [hunk 1]
```diff
@@ -9,10 +9,10 @@
     <tbody data-bind="foreach: rows">
     <tr class="header">
         <th class="code"></th>
-        <th class="outcome required">Outcome statement/s</th>
+        <th class="outcome required">${outcomeStatementHeading ?: 'Outcome Statement/s'}</th>
```
Comment (chrisala) [inline grails-app/views/project/meriPlan/_extendedBaselineMonitoring.gsp:12]: Looks like most of the other default headings are sentence case - should probably use sentence case here as well.

**Review by chrisala — CHANGES_REQUESTED** [review]

**Review by chrisala — CHANGES_REQUESTED** [review]
Please also look at and correct the "Read only MERI Plan" template as well as the Comparison template

File: grails-app/views/project/meriPlan/_extendedBaselineMonitoring.gsp (hunk 1 above)
Comment (salomon-j) [reply grails-app/views/project/meriPlan/_extendedBaselineMonitoring.gsp]: ready for retesting

File: grails-app/views/project/meriPlan/_outcomeStatements.gsp [hunk 2]
```diff
@@ -18,7 +18,7 @@
         </g:else>
         <th class="outcome required">${subtitle ?: ""} <g:if test="${helpText}"><fc:iconHelp html="true" container="body">${helpText}</fc:iconHelp></g:if> </th>
         <g:if test="${extendedOutcomes}">
-            <th class="investment-priority required">Investment Priority</th>
+            <th class="investment-priority required">${investmentPriorityHeading ?: 'Investment Priority'}</th>
```
[the remark this answers is not in the document]
Comment (salomon-j) [reply grails-app/views/project/meriPlan/_outcomeStatements.gsp]: ready for retesting

**Comment by salomon-j** [comment]
> Please also look at and correct the "Read only MERI Plan" template as well as the Comparison template

#3041 is also ready for testing, updates were pushed using the same feature branch

File: grails-app/views/project/meriPlanReadOnly/_monitoringIndicators.gsp [hunk 3]
```diff
@@ -5,13 +5,13 @@
     <thead>
     <tr>
         <th class="index"></th>
```
Comment (chrisala) [inline grails-app/views/project/meriPlanReadOnly/_monitoringIndicators.gsp:7]: This should be templated as per /meriPlan/_monitoringIndicators.gsp

File: grails-app/views/project/meriPlanChanges/_projectMethodology.gsp [hunk 4]
```diff
@@ -4,7 +4,7 @@
 <table class="table">
     <thead>
```
Comment (chrisala) [inline grails-app/views/project/meriPlanChanges/_projectMethodology.gsp:5]: This should be templated as per /meriPlan/_projectMethodology.gsp

File: grails-app/views/project/meriPlanChanges/_additionalOutcomes.gsp [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <table class="table secondary-outcome">
     <thead>
     <tr>
-        <th class="outcome-priority">Secondary outcome(s)</th>
```
Comment (chrisala) [inline grails-app/views/project/meriPlanChanges/_additionalOutcomes.gsp:5]: This should be templated as per /meriPlan/_addionalOutcomes.gsp

**Review by chrisala — CHANGES_REQUESTED** [review]
Many of the view changes made hardcode labels that match NHT but will then make RLP labels wrong.  The view files need to be updated to be templated to use the same variables as the editable version of the MERI plan to keep them in sync.

**Pull request merged**